### PR TITLE
Fix issues from the merging of the "Open All Log Files" command.

### PR DESCRIPTION
**Pull request by rgrunber**
See https://github.com/redhat-developer/vscode-java/pull/2375#pullrequestreview-1014460921 for reasoning.

- Make "java.open.serverLog" visible
- Properly hide the stdout, stderr & client logs

Signed-off-by: Roland Grunberg <rgrunber@redhat.com>

Although the titles won't be seen, I guess it makes sense to be consistent and add them back. Let me know if the new translations for the stdout/stderr make sense in `package.nls.zh.json`.

### CODE REVIEW

File: package.nls.zh.json [hunk 1]
```diff
@@ -2,6 +2,10 @@
 	"java.server.mode.switch": "切换到标准模式",
 	"java.projectConfiguration.update": "更新项目",
 	"java.project.import": "将 Java 项目导入工作区",
+	"java.open.serverLog": "打开 Java 语言服务器日志文件",
+	"java.open.serverStdoutLog": "打开 Java 语言服务器输出日志文件",
+	"java.open.serverStderrLog": "打开 Java 语言服务器错误日志文件",
+	"java.open.clientLog": "打开 Java 插件日志文件",
```
Comment (jdneo) [inline package.nls.zh.json:8]: 👍

File: package.json [hunk 2]
```diff
@@ -1086,6 +1087,18 @@
           "command": "java.project.import.command",
           "when": "javaLSReady"
         },
+        {
+          "command": "java.open.clientLog",
```
Comment (jdneo) [inline package.json:1091]: Is this hide on purpose? Currently it can be seen in command palette (`Open Java Extension Log File`)

Comment (rgrunber) [reply package.json]: Yes, I guess the goal was to reduce the amount of commands but if we keep the server logs, we may as well do client, so I have added it back.

Comment (jdneo) [reply package.json]: Got it.

**Pull request merged**